test/e2e/storage/external/external.go

```diff
@@ -39,6 +39,77 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
+// DriverDefinition needs to be filled in via a .yaml or .json
+// file. It's methods then implement the TestDriver interface, using
```

Comment:
typo nit: It's -> Its

Reply:
I was just moving the code, but I might as well fix it. Done.